File: src/main/java/me/yoursole/ctf/DataFiles/Items/Flag.java
package me.yoursole.ctf.DataFiles.Items;

import org.bukkit.ChatColor;
import org.bukkit.Material;
import org.bukkit.inventory.ItemFlag;
import org.bukkit.inventory.ItemStack;
import org.bukkit.inventory.meta.ItemMeta;

import java.util.ArrayList;
import java.util.List;

public class Flag {
    public static ItemStack flag;

    static {
        ItemStack flag = new ItemStack(Material.BLUE_BANNER);
        ItemMeta meta = flag.getItemMeta();

        List<String> lore = new ArrayList<>();
        meta.setDisplayName(ChatColor.BLUE+"The Flag");
        lore.add(ChatColor.GOLD+"Keep me safe");
        lore.add(ChatColor.GOLD+"When I am in your inventory you get points");
        meta.setLore(lore);
        meta.setUnbreakable(true);
        meta.setCustomModelData(1);
        meta.addItemFlags(ItemFlag.HIDE_ATTRIBUTES, ItemFlag.HIDE_UNBREAKABLE);
        flag.setItemMeta(meta);
        Flag.flag = flag;
    }
}

File: src/main/kotlin/me/yoursole/ctf/datafiles/items/Flag.kt
package me.yoursole.ctf.datafiles.items

import me.yoursole.ctf.datafiles.Utils.setCTFId
import org.bukkit.Material
import org.bukkit.craftbukkit.v1_17_R1.inventory.CraftItemStack
import org.bukkit.inventory.ItemFlag
import org.bukkit.inventory.ItemStack

object Flag {
    val flag: CraftItemStack by lazy {
        CraftItemStack.asCraftCopy(ItemStack(Material.BLUE_BANNER)).apply {
            itemMeta = itemMeta.apply {
                setDisplayName("§9The Flag")
                lore = listOf(
                    "§6Keep me safe",
                    "When I am in your inventory you get points"
                )
                isUnbreakable = true
                setCustomModelData(1)
                addItemFlags(ItemFlag.HIDE_ATTRIBUTES, ItemFlag.HIDE_UNBREAKABLE)
            }
            setCTFId("flag")
        }
    }
}